src/hosts/installer.rs

```diff
@@ -38,5 +39,30 @@ pub fn install_host(browser: &Browser) -> Result<PathBuf, &'static str> {
     }
     .map_err(|_| "Failed to write to browser host file.")?;
 
+    let os_type = get_os_type();
+    match (os_type, browser) {
+        (OsType::Windows, Browser::Firefox) => register_firefox(&full_write_path)?,
+        _ => (),
+    };
+
```

Comment:
For clarity, this means that Windows still needs the JSON manifest written somewhere, _and_ it additionally requires the registry entry?

Reply:
Yes. The registry key points to a file in the filesystem; I kept the file in the same location for simplicity.